pkg/services/apiserver/aggregator/aggregator.go

```diff
@@ -151,11 +151,7 @@ func CreateAggregatorConfig(commandOptions *options.Options, sharedConfig generi
 	if commandOptions.KubeAggregatorOptions.RemoteServicesFile == "" {
 		return NewConfig(aggregatorConfig, sharedInformerFactory, []builder.APIGroupBuilder{serviceAPIBuilder}, nil), nil
 	}
-
-	_, err = readCABundlePEM(commandOptions.KubeAggregatorOptions.APIServiceCABundleFile, commandOptions.ExtraOptions.DevMode)
```

Comment:
this change seems unrelated to the feature flag. why is it being removed?

Reply:
It was causing a crash when we first tried to omit setting the certs in HG, its not a big deal now since we are keeping the FF - but it is a noop until K8s fixes the issue upstream, so can delete it or keep it. What would you prefer?